PyStableMotifs/Reduction.py

```diff
@@ -514,6 +530,59 @@ def merge_source_motifs(self):
         for state in it.product([0,1],repeat=len(source_vars)):
             self.merged_source_motifs.append({v:x for v,x in zip(source_vars,state)})
 
+    def simple_merge_source_motifs(self,primes,MPBN_update=False):
+        """Merges stable motifs (and time-reversal stable motifs) that correspond to source nodes, e.g. A*=A, into combined motifs to
+        avoid combinatorial explosion. For example, A*=A, B*=B, C*=C produces six motifs that can stabilize in 8 ways; without
+        merging, these 8 combinations lead to 8*3!=48 successions because they can be considered in any order. This is silly because
+        source nodes all stabilize simultaneously.
+
+        Assumes that stable_motifs have already been computed,
+        but time_reverse_primes and time_reverse_stable_motifs are not.
+
+        To be used in the case of MBPN update.
+
+        Parameters
+        ----------
+        primes : dictionary of lists of lists of dictionaries
```

Comment:
Throughout the documentation, we refer to this data type as a "PyBoolNet primes dictionary", and you don't need to give an example here. So this should be something like this
```python
"""Merges stable motifs . . . 
Parameters
----------
primes : PyBoolNet primes dictionary
     PyBoolNet update rules whose source node stable motifs are to be merged.

Returns
. . .
"""
```

Reply:
I fixed it.
It was more like a note for myself.